data/overview.yaml

```
@@ -40,30 +66,67 @@ group:
     convolution: ''
     examples for groups with a small cardinal: ''
 
-linear algebra:
-  vector space: 'algebra/module.html#vector_space'
-  linear map: 'algebra/module.html#linear_map'
-  product of vector spaces: 'algebra/pi_instances.html#prod.module'
-  vector subspace: 'algebra/module.html#subspace'
-  range of a linear map: 'linear_algebra/basic.html#linear_map.range'
-  kernel of a linear map: 'linear_algebra/basic.html#linear_map.ker'
-  quotient space: 'linear_algebra/basic.html#submodule.quotient'
-  sum of subspaces: 'linear_algebra/basic.html#submodule.complete_lattice'
-  direct sum: ''
-  complement subspace: ''
-  linear independent: 'linear_algebra/basis.html#linear_independent'
-  generating sets: ''
-  algebra of endomorphisms of a vector space: ''
-  general linear group: 'linear_algebra/basic.html#linear_map.general_linear_group'
-  invariant subspace of an endomorphism: ''
-  eigenvalue: ''
-  eigenvector: ''
-  linear representation: ''
-  irreducible representation: ''
-  finite dimension representations:
-    examples: ''
-    Schur's lemma: ''
+3. 
+Rings: 
+  Unitary rings:
+  ring morphisms: 
+  subrings: 
+  The ring Zn on the integers: 
+  Ideal of a commutative ring: 
+  quotient rings: 
+  first ideals: 
+  maximal ideals: 
+  Chinese remainder theorem: 
+  Commutative ring algebra:
+  Divisibility in commutative integral rings: 
+  Irreducible elements: 
+  invertible elements: 
+  first elements among them: 
+  Factorial rings: 
+  Greatest common divisor: 
+  least common multiple:
+  Factorability of A[X] when A is a factorial ring:
+  Principle ring: 
+  Bezouts theorem: 
+  Euclidean rings: 
+  Euclids algorithm: 
+  Case of ring Z and K[X] algebra of polynomials on the field K: 
+  Irreducible polynomial: C
+  yclotomic polynomials in Q[X]: 
+  Einsteins criteria:
+  Congruence in Z: 
+  Prime numbers: 
+  Z/nZ and its invertible elements: 
+  Eurlers indicator function:
+Polynomial:
+  Polynomial algebra on one or several indeterminates on a commutative ring:
+  Roots of a polynomial:
+  multiplicity: 
+  Relationship between the coefficients and the roots of a split polynomial: 
+  Newtonian sums: 
+  Derived polynomials: 
+  Decomposition into sums of homogeneous polynomials: 
+  Symmetric polynomials:
+ Fields:
+  subfields: 
+  Characteristics: 
+  Frobenius morphisms: 
+  field extensions: 
+  integral ring fractional fields: 
+  Field Q of rational numbers: 
+  Field R of real numbers: 
+  Field C of complex numbers: 
+  dAlembert-Gauss theorem: 
+  Algebraic elements transcendentals: 
+  Algebraic extensions: 
+  Closed algebraic fields: 
+  Rupture and decomposition fields: 
+  Finite fields:
+  Rational fraction fields with one indeterminate on the field: 
+  Decomposition into simple elements: 
```

partial fraction decomposition

---

Isn't partial fraction decomposition an integration technique?

---

According to [Wikipedia](https://en.wikipedia.org/wiki/Partial_fraction_decomposition), it's rather the process where you take f/g and rewrite it as a sum of simpler elements (explaining the name in French), but in English, it's really just decomposition fractions into others ones.

---

Ah, ok. Makes sense. 
```suggestion
  Decomposing fractions into simple elements: 
```